openweathermap.py

```diff
@@ -13,109 +16,160 @@ def __init__(self, **kwargs):
-            temp_today_temp = temp_today['temp']
-            temp_today_temp_max = temp_today['temp_max']
-            temp_today_temp_min = temp_today['temp_min']
-
-            pressure_today = weather_today.get_pressure()
-            pressure_today_press = pressure_today['press']
-            pressure_today_sea_level = pressure_today['sea_level']
-
-            humidity_today = weather_today.get_humidity()
-
-            wind_today = weather_today.get_wind()
-            # disable temporarily this data. Will be fixed in the next release of pyown
-            # see: https://github.com/csparpa/pyowm/issues/177
-            try:
-                wind_today_deg = wind_today['deg']
-            except KeyError:
-                wind_today_deg = None
-            wind_today_speed = wind_today['speed']
-
-            snow_today = weather_today.get_snow()
-            rain_today = weather_today.get_rain()
-            clouds_coverage_today = weather_today.get_clouds()
-
-            message = {
-                "location": self.location,
-
-                "weather_today": weather_today_status,
-                "sunset_today_time": sunset_time_today,
-                "sunrise_today_time": sunrise_time_today,
-                "temp_today_temp": temp_today_temp,
-                "temp_today_temp_max": temp_today_temp_max,
-                "temp_today_temp_min": temp_today_temp_min,
-                "pressure_today_press": pressure_today_press,
-                "pressure_today_sea_level": pressure_today_sea_level,
-                "humidity_today": humidity_today,
-                "wind_today_deg": wind_today_deg,
-                "wind_today_speed": wind_today_speed,
-                "snow_today": snow_today,
-                "rain_today": rain_today,
-                "clouds_coverage_today": clouds_coverage_today,
-
-                "weather_tomorrow": weather_tomorrow_status,
-                "sunset_time_tomorrow": sunset_time_tomorrow,
-                "sunrise_time_tomorrow": sunrise_time_tomorrow,
-                "temp_tomorrow_temp": temp_tomorrow_temp,
-                "temp_tomorrow_temp_max": temp_tomorrow_temp_max,
-                "temp_tomorrow_temp_min": temp_tomorrow_temp_min,
-                "pressure_tomorrow_press": pressure_tomorrow_press,
-                "pressure_tomorrow_sea_level": pressure_tomorrow_sea_level,
-                "humidity_tomorrow": humidity_tomorrow,
-                # "wind_tomorrow_deg": wind_tomorrow_deg,
-                "wind_tomorrow_speed": wind_tomorrow_speed,
-                "snow_tomorrow": snow_tomorrow,
-                "rain_tomorrow": rain_tomorrow,
-                "clouds_coverage_tomorrow": clouds_coverage_tomorrow
-            }
-
-            self.say(message)
-
+                self.extended_location = self.location + "," + self.country
+            if not self.day:
+                owm = pyowm.OWM(API_key=self.api_key, language=self.lang)
+
+                # Tomorrow
```

Comment:
is this part still needed then? Is still working?

Reply:
Yes its still working, I just wanted to avoid to call the lib if only a forecast is called